Lib/glyphsLib/interpolation.py

```diff
@@ -40,6 +41,10 @@ def interpolate(ufos, master_dir, out_dir, instance_data,
         ufos, master_dir, out_dir, instance_data, italic)
 
     print('>>> Building instances')
+    # make sure old UFO data is removed (may contain deleted glyphs)
+    for path, _ in instance_files:
+        if os.path.exists(path):
+            shutil.rmtree(path)
```

Comment:
Umm.  This makes me a bit dizzy.  Can you please only do that if path ends with '.ufo'?